### PR TITLE
Remove velocity

### PR DESCRIPTION
**Pull request by daniel-heppner-ibigroup**
Velocity-react library is no longer maintained. I've replaced it with a very simple new library called react animate height that has types, hooks, and is being updated. https://www.npmjs.com/package/react-animate-height

### CODE REVIEW

**Comment by daniel-heppner-ibigroup** [comment]
Percy is catching the mysterious disappearing stop icons.

File: lib/components/viewers/RouteRow.js [hunk 1]
```diff
@@ -122,8 +123,14 @@ export class RouteRow extends PureComponent {
 
   render() {
     const { intl, isActive, operator, route } = this.props
+    const { newHeight } = this.state
     const { ModeIcon } = this.context
 
+    // Used to ensure details are visible until animation completes
+    const setNewHeight = (newHeight) => {
+      this.setState({ newHeight })
+    }
```
Comment (binh-dam-ibigroup) [inline lib/components/viewers/RouteRow.js:132]: Move this to a class method.

**Review by binh-dam-ibigroup — APPROVED** [review]
Just one small change but otherwise looks good.

**Review by miles-grant-ibigroup — APPROVED** [review]
No noticeable difference and bundle size is down (a tiny bit)

Thanks for the changes!

**Pull request merged**